src/sentry/models/groupcategory.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, Model
+
+
+class GroupCategory(Model):
+    __core__ = False
+
+    group = FlexibleForeignKey("sentry.Group")
+    category = models.TextField(null=False, db_index=True)
```

Comment:
This would probably be better off as an enum (store a small positive int instead), or maybe another model